docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -258,6 +266,10 @@ At this time, you cannot embed a checkout using a hosted payment gateway. See [A
 
 For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari).
 
+### How do I make sure that authenticated shoppers who sign out from the checkout page are also signed out of the headless storefront?
+
+To ensure you log shoppers off from the checkout page and the headless storefront, developers should pass an `onSignOut` option to `embedCheckout` to handle sign out events from the checkout page. There is not a corresponding event to handle logging in on the checkout page.
```

Comment:
"There is not a corresponding event to handle logging in on the checkout page." is true, but I'm not sure that we want to advertise that. Consider leaving that sentence out.